Makefile

```diff
@@ -22,11 +22,11 @@ build.platform.all: build.platform.darwin \
 
 .PHONY: build.platform.darwin
 build.platform.darwin:
-	@$(MAKE) build PLATFORM=darwin_amd64
+	@$(MAKE) build PLATFORM=darwin_amd64 FEATURES=pure
```

Comment:
i think you should add these here: https://github.com/mlab-lattice/lattice/blob/7a6e0926849e937485ce6634b11ca3b004faa254/ci/build-latticectl-binaries.sh#L24-L26